twitter/friends.go

```diff
@@ -0,0 +1,73 @@
+package twitter
+
+import (
+	"net/http"
+
+	"github.com/dghubble/sling"
+)
+
+// FriendIDs is a cursored collection of friend ids.
+type FriendIDs struct {
+	IDs               []int64 `json:"ids"`
+	NextCursor        int64   `json:"next_cursor"`
+	NextCursorStr     string  `json:"next_cursor_str"`
+	PreviousCursor    int64   `json:"previous_cursor"`
+	PreviousCursorStr string  `json:"previous_cursor_str"`
+}
+
+// Friends is a cursored collection of friends.
+type Friends struct {
+	Users             []User `json:"users"`
+	NextCursor        int64  `json:"next_cursor"`
+	NextCursorStr     string `json:"next_cursor_str"`
+	PreviousCursor    int64  `json:"previous_cursor"`
+	PreviousCursorStr string `json:"previous_cursor_str"`
+}
+
+// FriendService provides methods for accessing Twitter friends endpoints.
+type FriendService struct {
+	sling *sling.Sling
+}
+
+// newFriendService returns a new FriendService.
+func newFriendService(sling *sling.Sling) *FriendService {
+	return &FriendService{
+		sling: sling.Path("friends/"),
+	}
+}
+
+// FriendIDParams are the parameters for FriendService.Ids
+type FriendIDParams struct {
+	UserID     int64  `url:"user_id,omitempty"`
+	ScreenName string `url:"screen_name,omitempty"`
+	Cursor     int64  `url:"cursor,omitempty"`
+	Count      int    `url:"count,omitempty"`
+}
+
+// IDs returns a cursored collection of user ids following the specified user.
```

Comment:
The main point is that the comment is wrong. Those the specified user is following vs user ids following the user.

Yes, please break comment lines so they don't exceed 80 characters for consistency with this project and convenience with other tools.